nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/blob/binaries/BinariesObject.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.rest.management.blob.binaries;
+
+import static javax.ws.rs.core.Response.Status.CONFLICT;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+import org.nuxeo.ecm.core.blob.DocumentBlobManager;
+import org.nuxeo.ecm.core.blob.binary.BinaryManagerStatus;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.DefaultObject;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * Endpoint to manage the binaries.
+ *
+ * @since 11.1
+ */
+@WebObject(type = "binaries")
+public class BinariesObject extends DefaultObject {
+
+    /**
+     * Launches a garbage collect the unused binaries (orphaned).
+     * 
+     * @return {@link BinaryManagerStatus} if no gc is in progress, otherwise a
+     *         {@link javax.ws.rs.core.Response.Status#CONFLICT}
+     */
+    @POST
+    @Path("gc/orphaned")
+    public Response markAndSweep() {
+        DocumentBlobManager documentBlobManager = Framework.getService(DocumentBlobManager.class);
+
+        if (!documentBlobManager.isBinariesGarbageCollectionInProgress()) {
+            BinaryManagerStatus binaryManagerStatus = documentBlobManager.garbageCollectBinaries(true);
+            return Response.ok(binaryManagerStatus).build();
+        }
+
+        return Response.status(CONFLICT).build();
+    }
+
+    /**
+     * Checks if the gc on orphaned binaries is in progress.
+     *
+     * @return {@code true} if the gc is inProgress, {@code false} otherwise
+     */
+    @GET
+    @Path("gc/orphaned/inProgress")
+    public boolean isInProgress() {
```

Comment:
What's the point of this endpoint? Is it really needed? I don't think so.

Reply:
I added in a case where the Cloud team need to check / monitor if there any gc in progress. actually this is done in the JSF ui, we cannot launch a GC if there is one in progress.

Reply:
Well, the first endpoint returns a `409` if there is a GC in progress, I guess it's enough for now.
This endpoint was not in the specs, keep it simple - we'll see if needed later on.

Moreover, if we really want to have that kind of endpoint, it should not return a simple `true`/`false` string but a real JSON object anyway.